### PR TITLE
Handle the operator policy message changes

### PR DESCRIPTION
**Pull request by mprahl**
Specifically, the CSV name is now listed in the message.

### CODE REVIEW

**Review by JustinKuli — DISMISSED** [review]

**Comment by mprahl** [comment]
> /hold for WIP status

It's WIP since I didn't test it locally, so I'm letting CI validate it :laughing:

**Comment by mprahl** [comment]
This is blocked by https://github.com/open-cluster-management-io/config-policy-controller/pull/234.

**Comment by mprahl** [comment]
@JustinKuli this is ready for review now

**Review by JustinKuli — APPROVED** [review]

File: test/integration/compliance_history_test.go [hunk 1]
```diff
@@ -822,12 +832,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 
 				event0 := events[0].(map[string]interface{})
```
Comment (dhaiducek) [inline test/integration/compliance_history_test.go:833]: Is this guaranteed to have something in it?

**Pull request merged**